### PR TITLE
Auto readme updater

### PR DESCRIPTION
**Pull request by gnuhead-chieb**
Added Automatic readme updater.
Contributors doesn't need to update readme by them self anymore

### CODE REVIEW

**Comment by MrBrain295** [comment]
I feel that this would just create a bunch of unnecessary commits.

**Comment by gnuhead-chieb** [comment]
> I feel that this would just create a bunch of unnecessary commits.

this squash and rebase previous commit.
so it will not increase unnecessary commits

**Comment by gnuhead-chieb** [comment]
@MrBrain295 Ping

People often seem to make the mistake of updating the README manually instead of using the automatic updater.
By automating this, such mistakes can be avoided.

File: .github/workflows/update_readme.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Update Readme
+
+on:
+  push:
+    branches:
+      - master
```
Comment (MrBrain295) [inline .github/workflows/update_readme.yml:6]:
```suggestion
      - main
```

**Pull request merged**